package.json

```diff
@@ -14,6 +14,7 @@
     "e2e": "jest --config e2e.jest",
     "e2e-watch": "jest --watch --config e2e.jest",
     "clean": "rm -rf lib/*",
+    "start": "babel src -w -d lib",
```

Comment:
I would call this `watch` instead of `start`, as it's not really "starting" an app.